src/packages/apollo/src/plugins/graphql-deduplicator.ts

```diff
@@ -0,0 +1,21 @@
+import { ApolloServerPlugin } from '@apollo/server';
+import { deflate } from 'graphql-deduplicator';
+
+export const dedupeGraphQL: ApolloServerPlugin = {
+	async requestDidStart() {
+		return {
+			async willSendResponse(requestContext) {
+				if (
+					requestContext.response.body.kind == 'single' &&
+					requestContext.response.body.singleResult.data
+				) {
+					if (typeof requestContext.response.body.singleResult.data.result === 'object') {
```

Comment:
is this to check if its an Array?

if so, you could use:

`if (Array.isArray(requestContext.response.body.singleResult.data.result)) {`